forge/src/executor/abi.rs

```diff
@@ -100,20 +30,21 @@ ethers::contract::abigen!(
             event log_named_uint         (string key, uint val)
             event log_named_bytes        (string key, bytes val)
             event log_named_string       (string key, string val)
-            ]"#,
+    ]"#
 );
-pub use hevmconsole_mod::HEVMCONSOLE_ABI;
-
-// Bindings for hardhat console
-ethers::contract::abigen!(Console, "./testdata/console.json",);
 pub use console_mod::CONSOLE_ABI;
 
+// Bindings for Hardhat console
+ethers::contract::abigen!(HardhatConsole, "./abi/console.json",);
+pub use hardhatconsole_mod::HARDHATCONSOLE_ABI as HARDHAT_CONSOLE_ABI;
```

Comment:
We only use this to decode the calls themselves, but in the `LogCollector` I actually cheat a little bit to avoid returning a `Vec<String>` for logs (since we agreed to move that logic to the CLI) by decoding the inputs, converting them to a string and pretending like a `log(string)` event was emitted